determined-ai/determined
# fix: double scroll bars in chart grid

## Description
[WEB-932]

https://user-images.githubusercontent.com/109113616/227026615-5e9b55ca-7ec0-4b04-a593-1c7d0cbb1bb6.mov





## Test Plan
- Go to experiment overview page
- Enable chart flag by adding `f_chart=on` if we still use chart flag
- Narrow screen horizontally
- Ideally there are at least 3 charts
- Check if no double scroll bars even if screen width is changed





## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket






[WEB-932]: https://hpe-aiatscale.atlassian.net/browse/WEB-932?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
